app/views/katello/api/v2/content_views/base.json.rabl

```diff
@@ -12,6 +12,7 @@ attributes :import_only
 attributes :generated_for
 attributes :related_cv_count
 attributes :related_composite_cvs
+attributes :needs_publish?
```

Comment:
By convention the rabl attributes don't have question marks, just the Ruby methods. examples: `katello_tracer_installed`, `katello_agent_enabled`